### DISCUSSION TITLE
Translate?

### DISCUSSION THREAD

**Discussion by rex07** [NeoApplications/Neo-Store#81] [Ideas]
Add crowdin or weblate for translation.

**Comment by Iamlooker** [comment]
I just created the the project on crowdin

https://crowdin.com/project/droid-ify

**Comment by comradekingu** [comment]
@iamlooker
Please delete the project on Crowdin.
If it is integrated it means my contributions are uploaded there, which I never agreed to.
Moreover, the broken locale management in Crowdin will prevent my translation from showing up.

There are 10 translations on https://hosted.weblate.org/projects/droidify/localization/, vs. 3.5 on Crowdin, in a third of the time…

**Comment by comradekingu** [comment]
@rex07 You can translate Droid-ify on https://hosted.weblate.org/projects/droidify/localization/ now :)
